resolwe_bio/processes/plots/bamliquidator.yml

```diff
@@ -0,0 +1,185 @@
+# =============
+# Bamliquidator
+# =============
+---
+
+- slug: bamliquidator
+  name: Bamliquidator
+  requirements:
+    expression-engine: jinja
+    executor:
+      docker:
+        image: "resolwe/bio-linux8-resolwe-preview:bamliquidator"
+  data_name: "Bamliquidator"
+  version: 0.0.8
+  type: data:bam:plot:bamliquidator
+  category: analyses
+  persistence: CACHED
+  description: >
+    Set of tools for analyzing the density of short DNA sequence read alignments in the BAM file format.
+  input:
+    - name: analysis_type
+      label: Analysis type
+      type: basic:string
+      default: bin
+      choices:
+        - label: Bin mode
+          value: bin
+        - label: Region mode
+          value: region
+        - label: BED mode
+          value: bed
+    - name: bam
+      label: BAM File
+      type: list:data:alignment:bam
+    - name: cell_type
+      label: Cell type
+      type: basic:string
+      default: cell_type
+    - name: bin_size
+      label: Bin size
+      type: basic:integer
+      hidden: "analysis_type != 'bin'"
+      required: False
+      description: >
+        Number of base pairs in each bin. The smaller the bin size the longer the runtime and the larger the data files.
+        Default is 100000.
+    - name: regions_file_gtf
```

Comment:
I think that `file` is redundant here, it would be better to use just `regions_gtf`